orttraining/orttraining/eager/opgen/opgen/atenops.py

```diff
@@ -173,6 +173,8 @@ def __init__(self, dY, X):
     "aten::nonzero": Transpose(NonZero("self")),
     "aten::nonzero.out": SignatureOnly(),
     "aten::_log_softmax.out": SignatureOnly(),
+    #  nll_loss_forward.output can't be implemented due to the unsupported ONNXOp in the CPU Execution Provider
```

Comment:
nit: `NegativeLogLikelihoodLoss is not supported by the CPU Execution Provider so testing is not possible. Leaving nll_loss_forward.output set to fallback.` Keep the link to the operatorKernels.md (delete the `)` at the end)

Just a suggestion to be a little clearer on the onnx op (NegativeLogLikelihoodLoss) versus the aten method (nll_loss_forward.output).